docs/_snippets/flutter_infinite_list_tutorial/post_bloc.dart.md

```diff
@@ -0,0 +1,74 @@
+```dart
```

Comment:
can we remove this snippet and just point to the actual file in github to ensure it stays up to date? Thanks!